README.md

```diff
@@ -4,4 +4,4 @@ The list below is the names of students and participants in my professional cour
 
 | First Name      | Last Name | College | Courses | Social Media | Comments |
 | :----: | :----: | :----: | :----: | :----: | :----: |
-| Example     | Example     | - | PHP, HTML, CSS | [Telegram](https://t.me) | My dude |
+| Milad     | Takjoo     | - | PHP, HTML, CSS | [Telegram](https://t.me) | m |
```

Comment:
- Your correct college abbr is 'MFT'
- Set correct Courses name : { HTML,CSS,JS,GIT }
- If you don't have any social media use dash ( - )
- Leave the comment blank or use meaningful sentences